sources/packages/api/src/route-controllers/student/student.controller.service.ts

```diff
@@ -122,4 +134,50 @@ export class StudentControllerService {
       validSin: student.sinValidation.isValidSIN,
     };
   }
+
+  /**
+   * API to fetch all the applications that belong to student.
```

Comment:
The comment says API. It may be removed as controller service is not an API by itself unless used by a controller.

Reply:
It was copied as it was and I particularly don't like these comments also because they are not accurate. I will change it.

Reply:
Done